core-api/src/main/java/com/optimizely/ab/optimizelyconfig/OptimizelyConfig.java

```diff
@@ -46,14 +59,20 @@ public String getRevision() {
         return revision;
     }
 
+    @JsonIgnore
+    public String getDatafile() {
+        return datafile;
+    }
+
     @Override
     public boolean equals(Object obj) {
         if (obj == null || getClass() != obj.getClass()) return false;
         if (obj == this) return true;
         OptimizelyConfig optimizelyConfig = (OptimizelyConfig) obj;
         return revision.equals(optimizelyConfig.getRevision()) &&
             experimentsMap.equals(optimizelyConfig.getExperimentsMap()) &&
-            featuresMap.equals(optimizelyConfig.getFeaturesMap());
+            featuresMap.equals(optimizelyConfig.getFeaturesMap()) &&
+            Objects.equals(datafile, optimizelyConfig.getDatafile());
```

Comment:
Do we need include "datafile" string compare?  String with formatting spaces, etc. will make compare fal. 
It should be enough to compare the rest of fields for OptimizelyConfig equality.